gate_names: ['Gate10', 'Gate21', 'Gate2', 'Gate13', 'Gate9', 'Gate14', 'Gate1', 'Gate22', 'Gate15', 'Gate23', 'Gate6']
uav:
  flightgoggles_uav_dynamics:
    init_pose: [18.0, -23.0, 5.3, 0.0, 0.0, 0.707, 0.707]


